src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -159,9 +159,10 @@ impl CoverageTask {
 
             context.heartbeat.alive();
         }
+        info!("report initial coverage");
+        context.report_coverage_stats().await;
```

Comment:
we should report this before we even process the readonly_inputs

Reply:
Done.